### ISSUE TITLE
It does not solve all types of hCaptcha

### ISSUE THREAD

**Issue by tommasocarraro** [NopeCHALLC/nopecha-extension#38]
Hi all.

I am using this extension in a small scraping project.

I found the captcha in the image, and this tool cannot solve it. The tool presses in a random location of the picture.

Thank you for your help!

I found it can solve classic grid hCaptchas, but this type.

<img width="506" alt="Screenshot 2024-01-16 at 17 40 20" src="https://github.com/NopeCHALLC/nopecha-extension/assets/34162319/e3c44f51-91df-4891-92f1-18f7641bed86">

**Comment by kiennd** [comment]
i got the same problem with recaptcha v3

**Comment by tommasocarraro** [comment]
As this tool is not able to solve all kind of CAPTCHAs, there are better free alternatives.

I found [this](https://chromewebstore.google.com/detail/captcha-solver-free-auto/hlifkpholllijblknnmbfagnkjneagid?hl=en-US) totally free extension that works really well. In particular, it infinitely tries till success. 

Despite that, this issue will stay open until a solution is provided. I also tried to get in touch with the developers in the given Discord channel, but no answers in weeks.
